lib/Controller/AlbumsController.php

```diff
@@ -109,13 +110,36 @@ private function formatData(iterable $nodes): array {
 				'mime' => $node->getMimetype(),
 				'size' => $node->getSize(),
 				'type' => $node->getType(),
+				'permissions' => $this->formatPermissions($node->getPermissions()),
```

Comment:
Ah, and this is not the albums controller but the folders controller, probably, then we may need this for the actual albums as well

Reply:
Yep, but the other Views are importing `src/services/DavRequest.js`, so they will benefit from that too

Reply:
https://github.com/nextcloud/photos/search?q=DavRequest

Reply:
Yes, but Albums and Faces dav endpoints do not expose permissions in DAV afaik